monitoring/uss_qualifier/webapp/docker-compose-webapp.yaml

```diff
@@ -0,0 +1,86 @@
+version: '3.9'
+services:
+  mock-riddp:
```

Comment:
seeing attached messages in the logs, not sure how relevant these are w.r.t docker-compose config.
<img width="698" alt="Screen Shot 2022-05-26 at 11 20 23 pm" src="https://user-images.githubusercontent.com/40968555/170496025-25d024ab-0c43-4694-9cff-1e783469b8ec.png">